hindsight/ConnectFlow/UI/LoginViewController.swift

```diff
@@ -14,17 +14,37 @@ import RxSwift
 
 class LoginViewController: UIViewController {
 
-    lazy var facebookConnect: UIButton = {
-        let button = UIButton()
-        button.setTitle("Connect With Facebook", for: .normal)
-        button.backgroundColor = ColorName.hindsightBlue.color
-        return button
-    }()
-
     let viewModel: LoginViewModelProtocol
 
     let disposeBag = DisposeBag()
 
+    lazy var logoImageView: UIImageView = {
+        let view = UIImageView(frame: .zero)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    lazy var backgroundImageView: UIImageView = {
+        let view = UIImageView(frame: .zero)
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    lazy var facebookConnect: HindsightLoginButton = {
+        let normalState = LoginButtonState(text: "Login With Facebook",
+                                           textColor: ColorName.hindsightWhite.color,
+                                           backgroundColor: ColorName.facebookNormalBlueColor,
+                                           image: UIImage(named: "fb_logo"))
```

Comment:
@manishb24 can image be part of your script as well?

Reply:
yes, We have updated the project to generate the images from Swiftgen.

Reply:
I have updated the code to use images from Images.swift which generated by the SwiftGen.